policy/modules/services/xserver.te

```diff
@@ -59,10 +59,10 @@ gen_tunable(allow_write_xshm, false)
 ## <desc>
 ## <p>
 ## Allows clients to write to the X server tmpfs
-## files.
+## files (needed by the Qt library).
 ## </p>
 ## </desc>
-gen_tunable(xserver_client_writes_xserver_tmpfs, false)
+gen_tunable(xserver_client_writes_xserver_tmpfs, true)
```

Comment:
Please do not change this default.

Reply:
Yes, considering https://github.com/SELinuxProject/refpolicy/pull/907#discussion_r2093692605, I agree and will amend both the default value and comment.